tsn-vpn-over-mpls/draft-ietf-detnet-tsn-vpn-over-mpls.xml

```diff
@@ -351,108 +355,138 @@ Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
+                support the TSN Stream identification functions and the related
+                managed objects as defined in <xref target="IEEE8021CB">IEEE
+                802.1CB</xref> and <xref target="IEEEP8021CBdb">IEEE
+                P802.1CBdb</xref> to recognize the App-flow related
+                packets. 
     </t>
     <t>
-		Implementations of this document SHALL use management and
-		control information to map a TSN Stream to a DetNet flow. 
-		N:1 mapping (aggregating TSN Streams in a single
-		DetNet flow) SHALL be supported. The management or control
-		function that provisions flow mapping SHALL ensure that
-		adequate resources are allocated and configured to provide
-		proper service requirements of the mapped flows.
-	</t>
-	<t>
-		Due to the (intentional) similarities of the DetNet PREOF and TSN FRER 
-		functions service protection function interworking is possible between 
-		the TSN and the DetNet domains. Such service protection interworking 
-		scenarios MAY require to copy sequence number fields from TSN (L2) to 
-		PW (MPLS) encapsulation. However, such interworking is out-of-scope in 
-		this document and left for further study.
-	</t>
+                Implementations of this document SHALL use management and
+                control information to map a TSN Stream to a DetNet flow. 
+                N:1 mapping (aggregating TSN Streams in a single
+                DetNet flow) SHALL be supported. The management or control
+                function that provisions flow mapping SHALL ensure that
+                adequate resources are allocated and configured to provide
+                proper service requirements of the mapped flows. -->
+        </t>
+        <t>
+                Due to the (intentional) similarities of the DetNet PREOF and
+                TSN FRER functions service protection function interworking is
+                possible between the TSN and the DetNet domains. Such service
+                protection interworking scenarios MAY require to copy sequence
+                number fields from TSN (L2) to PW (MPLS) encapsulation.
+                However, such interworking is out-of-scope in this document and
+                left for further study.
+        </t>
+        <t>
+                The Service proxy SHALL support TSN MAC addresses allowing a
+                TSN connection to connect to another TSN network over the
+                DetNet Network. A service proxy SHALL be configured as either a
+                unicast DetNet application or a multicast application.  Service
+                proxies that support unicast service SHALL support TSN unicast
+                MAC addresses and Service proxies that support multicast
+                services SHALL support TSN multicast addresses.  The TSN
+                specific functions on the DetNet Edge Node SHOULD participate
+                in TSN Stream reservation protocols allowing TSN to establish

```

Comment:
Reservation protocols we do not intend to cover here. Also, they are not used in fully centralized model. Let's discuss this whole paragraph, whether we need this new text.